src/kernels/Conv_Winograd_v21_1_2_metadata.inc

```diff
@@ -71,28 +71,32 @@ amdhsa.kernels:
     - { .size: 4, .offset:  84, .value_kind: by_value, .value_type: i32, .name: out_w }
     - { .size: 8, .offset:  88, .value_kind: global_buffer, .value_type: f32, .name: bias_addr,    .address_space: global, .is_const: true }
     - { .size: 4, .offset:  96, .value_kind: by_value, .value_type: f32, .name: RELU_alpha }
-    - { .size: 4, .offset: 100, .value_kind: by_value, .value_type: i32, .name: d_N_stride }
-    - { .size: 4, .offset: 104, .value_kind: by_value, .value_type: i32, .name: d_C_stride }
-    - { .size: 4, .offset: 108, .value_kind: by_value, .value_type: i32, .name: d_H_stride }
-    - { .size: 4, .offset: 112, .value_kind: by_value, .value_type: i32, .name: d_W_stride }
-    - { .size: 4, .offset: 116, .value_kind: by_value, .value_type: i32, .name: f_K_stride }
-    - { .size: 4, .offset: 120, .value_kind: by_value, .value_type: i32, .name: f_C_stride }
-    - { .size: 4, .offset: 124, .value_kind: by_value, .value_type: i32, .name: f_R_stride }
-    - { .size: 4, .offset: 128, .value_kind: by_value, .value_type: i32, .name: f_S_stride }
-    - { .size: 4, .offset: 132, .value_kind: by_value, .value_type: i32, .name: o_N_stride }
-    - { .size: 4, .offset: 136, .value_kind: by_value, .value_type: i32, .name: o_K_stride }
-    - { .size: 4, .offset: 140, .value_kind: by_value, .value_type: i32, .name: o_H_stride }
-    - { .size: 4, .offset: 144, .value_kind: by_value, .value_type: i32, .name: o_W_stride }
-    - { .size: 4, .offset: 148, .value_kind: by_value, .value_type: i32, .name: G }
-    - { .size: 4, .offset: 152, .value_kind: by_value, .value_type: i32, .name: d_G_stride }
-    - { .size: 4, .offset: 156, .value_kind: by_value, .value_type: i32, .name: f_G_stride }
-    - { .size: 4, .offset: 160, .value_kind: by_value, .value_type: i32, .name: o_G_stride }
-    - { .size: 8, .offset: 168, .value_kind: hidden_global_offset_x, .value_type: i64 }
-    - { .size: 8, .offset: 176, .value_kind: hidden_global_offset_y, .value_type: i64 }
-    - { .size: 8, .offset: 184, .value_kind: hidden_global_offset_z, .value_type: i64 }
-    - { .size: 8, .offset: 192, .value_kind: hidden_none,   .value_type: i8 }
-    - { .size: 8, .offset: 200, .value_kind: hidden_none,   .value_type: i8 }
-    - { .size: 8, .offset: 208, .value_kind: hidden_none,   .value_type: i8 }
+    - { .size: 8, .offset: 104, .value_kind: by_value, .value_type: i64, .name: d_offset }
+    - { .size: 8, .offset: 112, .value_kind: by_value, .value_type: i64, .name: f_offset }
```

Comment:
Please explicitly add reserved2 field. See latest updates of the winograd shader.